PD-273 Reorganize and Update Support Articles (#1815)
truenas/documentation: content/SCALE/GettingStarted/Getting Support/GetSupportSCALE.md

---
title: "Accessing TrueNAS Support"
description: "This article provides instructions for SCALE users to access the TrueNAS Community and Social Media, get system support, and report problems though the SCALE UI."
weight: 10
aliases:
 - /scale/scaletutorials/systemsettings/general/getsupportscale/
tags:
 - scalesupport
---

{{< toc >}}

## About TrueNAS Support

{{< enterprise >}}
Enterprise SCALE hardware customers with support contracts should contact iXsystems Support using either the **[Commercial Support](https://www.truenas.com/commercial-support/)** option on the top header of the TrueNAS Documentation Hub website, or through one of the contact options listed below.
{{< expand "Contacting iXsystems Support" "v" >}}
{{< include file="static/includes/General/iXsystemsSupportContact.html.part" html="true" >}}
{{< /expand >}}
{{< /enterprise >}}

> TrueNAS SCALE users are welcome to report bugs, suggest new TrueNAS features, and vote for suggested improvment in the Jira project instance. 

Reviewer: ```suggestion
TrueNAS SCALE users are welcome to report bugs, suggest new TrueNAS features, and vote for suggested improvements in the Jira project instance. 
```

Have questions? We recommend searching through the software documentation and community resources for answers. 

> Non-Enterprise SCALE customers experiencing software bugs or instability can try to find answers in the various community forums or can [file an issue ticket](https://www.truenas.com/docs/contributing/issuereporting/) through the Jira ticket reporting system for TrueNAS. The [Issue Reporting]({{< relref "/content/Contributing/IssueReporting/_index.md" >}}) article for more information what type of information to include in issue tickets in Jira, and attaching files to issue tickets.

Reviewer: ```suggestion
Non-Enterprise SCALE customers experiencing software bugs or instability can try to find answers in the various community forums, or they can [file an issue ticket](https://www.truenas.com/docs/contributing/issuereporting/) through the Jira ticket reporting system for TrueNAS. The [Issue Reporting]({{< relref "/content/Contributing/IssueReporting/_index.md" >}}) article has more information on what type of information and attachments to include in Jira tickets.
```

This captures the system configuration information and logs iXsystems needs to help resolve your issues. 

{{< include file="/content/_includes/CreateDebugSCALE.md" type="page" >}}

Upload this debug to the private attachments area using the link provided when you open a Jira ticket. 
After uploading the file, link the attachment to the Jira ticket number before you click **Save**. 

Support is also available through the TrueNAS community forums, blog, and Discord. 
These options are accessible on the top header of the TrueNAS Documentation Hub website and from the links at the bottom of all articles.

## Using the TrueNAS Community
The [TrueNAS Community](https://www.truenas.com/community/) is an active online resource for asking questions, troubleshooting issues, and sharing information with other TrueNAS users.
You must [register](https://www.truenas.com/community/register/) to post.
We encourage new users to briefly [introduce](https://www.truenas.com/community/forums/introductions.25/) themselves and review the [forum rules](https://www.truenas.com/community/threads/forum-rules.45124/) before posting.

> [Community Resources](https://www.truenas.com/community/resources/) are user-contributed articles about every facet of using TrueNAS, organized into broad categories,and they incorporate a community rating system to better highlight content that the whole community has found helpful.

Reviewer: ```suggestion
[Community Resources](https://www.truenas.com/community/resources/) are user-contributed articles about every facet of using TrueNAS organized into broad categories. They incorporate a community rating system to better highlight content that the whole community has found helpful.
```
